CHANGELOG.md

```diff
@@ -1,11 +1,43 @@
 ## Unreleased
 
+## v0.23.1
+
+FEATURES:
```

Comment:
At least on the Foundations side, we would consider something like this an Improvement, not a Feature. Features generally being larger things that would be worthy of a blog post or some marketing.

It seems like in an older release of this plugin the Improvements section had "Adds abort_on_error parameter to CLI login command to help in non-interactive contexts" which sounds like a similar level of change.